pkg/plugins/install.go

```diff
@@ -25,8 +30,9 @@ func (o *InstallOptions) Validate(args []string, cxt *portercontext.Context) err
 			return fmt.Errorf("plugin URL should not be specified when --file is provided")
 		}
 
-		if o.Version != "" {
-			return fmt.Errorf("plugin version should not be specified when --file is provided")
+		// version should not be set to anything other than the default value
+		if o.Version != "" && o.Version != "latest" {
```

Comment:
When --version is empty, it defaults to latest so I'm not sure I get what this is checking for or guarding against? The error message just says that they can't specify --version at all so can't we just stick with the original check that version was specified?

Reply:
When no version is specified by users, the `o.Version` is default to `latest`. The original check would fail bc the `o.Version` is not empty.
The check here is to guard that users didn't try to set a version value through the flag. `o.Version` should not have value other than "latest" or empty

Reply:
Thanks! I didn't realize that we were defaulting to "latest" so early, I'm assuming it's the flag default. Makes sense! 👍